### ISSUE TITLE
I think you should make the script not require that user 'pi' install it

### ISSUE THREAD

**Issue by pwillard** [mrgw454/CoCo-Pi-Installer#1]
Since the Raspberry Pi organization has been recommending that we not use the default 'pi' account when building images, we should have a way in the setup script to allow for a user other than "pi"

**Comment by n6il** [comment]
I don't think this is strictly required.  I was able to install it on my pi using my normal user instead of pi... seems like things mostly work properly to me

**Comment by mrgw454** [comment]
I should have replied to this sooner (apologies).  As n6il has mentioned above, there really is no requirement for a specific user name to be used.  That being said, I will double-check scripts to confirm.
